pages/[network]/[chain]/bounty.tsx

```diff
@@ -81,45 +81,15 @@ export default function PageIssue({ bounty }: PageBountyProps) {
     setIsEditIssue(false)
   }
 
-  function checkForks(){
-    if (!state.Service?.network?.repos?.active?.githubPath || isRepoForked !== undefined) return;
-
-    if (bounty?.data?.working?.includes(state.currentUser?.login))
-      return setIsRepoForked(true);
-
-    const [, activeName] = state.Service.network.repos.active.githubPath.split("/");
-  
-    getUserRepository(state.currentUser?.login, activeName)
-    .then(repository => {
-      const { isFork, nameWithOwner, parent } = repository;
-
-      setIsRepoForked(isFork && parent?.nameWithOwner === state.Service.network.repos.active.githubPath ||
-        nameWithOwner.startsWith(`${state.currentUser?.login}/`));
-    })
-    .catch((e) => {
-      setIsRepoForked(false);
-      console.log("Failed to get users repositories: ", e);
-    });
-  }
-
   function addNewComment(comment) {
     setCommentsIssue([...commentsIssue, comment]);
   }
-  
+
   useEffect(() => {
-    if (!state.currentUser?.login ||
-        !state.currentUser?.walletAddress ||
-        !state.Service?.network?.repos?.active ||
-        !currentBounty?.data ||
-        isRepoForked !== undefined) 
-      return;
-    checkForks();
-  },[
-    state.currentUser?.login, 
-    currentBounty?.data?.working, 
-    state.Service?.network?.repos?.active,
-    !state.currentUser?.walletAddress
-  ]);
+    if(!state.currentUser?.walletAddress) return;
```

Comment:
this should be returned from the user session, since the user signed it; but it should also be left to the page-actions-controller to get this